### ISSUE TITLE
zypper se doesn't need root privileges

### ISSUE THREAD

**Issue by victorhck** [somebody1234/plugin-zypper#2]
Hello!
seems that sudo for zse (sudo zypper se) is not necesary (and a bit annonying :) )
Maybe can be removed
Thanks for your code!! :+1:

**Comment by corvore** [comment]
Yeah, I agree. I love the addon, but for zypper search it'd be nice if it dropped the sudo.

**Comment by victorhck** [comment]
I made this tweaks in my system, it's more easy and comfortable in that way!! :)

**Comment by corvore** [comment]
> I made this tweaks in my system, it's more easy and comfortable in that way!! :)

Same. I went ahead and modified those on my local install. :)

**Comment by corvore** [comment]
went ahead and did a pull request with my changes. I'm sure other functions could have sudo removed as well, but at least those two commonly used ones could use it.
